.releaserc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "branches": ["master"]
```

Comment:
Didn't you change it to main?

Reply:
Nope. It was created as main (PC) but I've changed it to master